experimental/dds/tree2/src/codec/README.md

```diff
@@ -0,0 +1,15 @@
+# Codec
+
+This library defines types and constructs for organizing code that works with persisted data.
+Its fundamental building block is a codec--a bit of code capable of transcoding between some encoded and in-memory format.
```

Comment:
Its possible to parse this sentence wrong, and since both formats are used in memory, its also not super clear.

```suggestion
Its fundamental building block is a codec--a bit of code capable of transcoding between a stable format used for persistence and a potentially less stable format only used in memory.
```